### PR TITLE
Move topics to dashboards page + dashboards page initial updates

### PR DESCRIPTION
**Pull request by dancer2090**
## Overview
This PR:
 - adds new /dashboards page with content like on 'topics page'. (we still use topics components. They will be renamed in future)
 - adds new routes /dashboards/:id are equal to /topics/:id
 - updates routes from /topics to /dashboards
 - adds "more" option to header-dropdown menu
 - adds new Hero text and new section texts,
 - adds gallery prototype to /dashboards from https://resourcewatch.org/data/dashboards (Visuals will update in future PR)
 - removed /data/dashboards route from the project. /data/dashboards/:slug for custom dashboards work.
## Testing instructions
Use staging API. Go to http://localhost:9000/dashboards or click dashboards on the header-menu.
Check all texts on this page.
Click on featured dashboard item to check /dashboards/:id page.
Click on gallery item or follow http://localhost:9000/data/dashboards/testing-2-c0f6e3a1-b4e3-4999-aec2-49137e0cd2ae
Follow http://localhost:9000/data/dashboards to get 404 page (removed)

## Pivotal tasks
[Menu title and routes](https://www.pivotaltracker.com/n/projects/1374154/stories/169732634)
[Hero text and first section text](https://www.pivotaltracker.com/n/projects/1374154/stories/169732737)
[More oprion](https://www.pivotaltracker.com/n/projects/1374154/stories/169732869)
[Remove unnecessary route](https://www.pivotaltracker.com/n/projects/1374154/stories/169733032)
[Add new "Dashboards gallery" section on the "Dashboards" page.](https://www.pivotaltracker.com/n/projects/1374154/stories/169732990)

---

## Checklist before submitting
- [ ] Title: Don't forget to make it clear for everyone, even for people that don't know about the feature/bug.
- [ ] Meaningful commits and code rebased on `develop`.

### CODE REVIEW

File: css/components/dashboards/dashboards-subheader-block.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.c-dashboards-subheader-block {
+  padding-bottom: 45px;
+  padding-top: 70px;
```
Comment (pablopareja) [inline css/components/dashboards/dashboards-subheader-block.scss]: @dancer2090 Better to use for paddings, margins, etc... a factor of the variable `$space`, e.g. `$space * 14` in the case of this line

File: layout/footer/constants.js [hunk 2]
```diff
@@ -15,28 +15,6 @@ export const FOOTER_LINKS = [
...
-    children: [
-      { label: 'Cities', route: 'topics_detail', params: { id: 'cities' } },
-      { label: 'Climate', route: 'topics_detail', params: { id: 'climate' } },
-
-      { label: 'Energy', route: 'topics_detail', params: { id: 'energy' } },
-      { label: 'Food', route: 'topics_detail', params: { id: 'food' } },
-      { label: 'Forests', route: 'topics_detail', params: { id: 'forests' } },
-      { label: 'Society', route: 'topics_detail', params: { id: 'society' } },
-      { label: 'Oceans', route: 'topics_detail', params: { id: 'oceans' } },
-      { label: 'Water', route: 'topics_detail', params: { id: 'water' } }
```
Comment (pablopareja) [inline layout/footer/constants.js]: @dancer2090 We should keep the links to the dashboards in the footer as well.

File: css/components/dashboards/dashboards-subheader-block.scss (hunk 1 above)
Comment (dancer2090) [reply css/components/dashboards/dashboards-subheader-block.scss]: Fixed https://github.com/resource-watch/resource-watch/pull/1179/commits/b044ec7607c66fcb8d92500e2fa4f7d7d57dec2d

**Comment by dancer2090** [comment]
> Here are my comments:
> 
> 1. We are missing the links to the topics/dashboards that currently exist in the footer, (we should also add the "more" link specified in one of the PT tasks mentioned in this PR).
>    image
> 2. According to Emily's comment in the corresponding PT task, the title for the dashboard gallery section should be in singular instead of plural
>    image
> 3. Why are the cards from the dashboard gallery section smaller in size than the ones from the _Featured dashboards_ section? According to the [design prototype](https://projects.invisionapp.com/d/main?origin=v7#/console/11337456/243102360/preview), they should have the same size in both sections.

@pablopareja 
1) Waiting for confirmation from Simao
2) Fixed https://github.com/resource-watch/resource-watch/pull/1179/commits/d6aad78e59dff4ca967cd97951df01562acf9446
3) In current PR we just move this gallery from /data/dashboards route, that was removed. The visual state will update in future PR.

**Comment by pablopareja** [comment]
Cool, thanks for fixing those details.
Good job with this PR! 💪 
Let's wait for @simaob confirmation about the only pending item before being able to merge this PR.

File: layout/footer/constants.js (hunk 2 above)
Comment (dancer2090) [reply layout/footer/constants.js]: Confirmed by daniel.caso:
The footer on the design is a bit outdated. Let’s keep it as it is now, keeping the featured dashboads listed.
We might consider to change it if we start including several more featured dashboards.

Fixed here: https://github.com/resource-watch/resource-watch/pull/1179/commits/1dce21a59d944c0e4c75ab45a95b414ffcf43e65

**Comment by dancer2090** [comment]
@pablopareja got confirmation from Daniel.caso. Fixed. 
It looks like everything ready for merging.

**Comment by pablopareja** [comment]
@dancer2090 I was testing the updates that you made in your last commit, and there's only one last thing that we should change before merging this PR.
The elements that appear in the `Dashboards` section from the footer component should be loaded dynamically, similarly to what we have in the navigation bar. 
Otherwise, when you test it, e.g. in staging, you'll have links pointing to pages that don't exist. Moreover, every time that one of the featured dashboards is renamed, removed, or created, the code would have to be updated, which isn't the right approach.

We're almost there! 💪

**Comment by dancer2090** [comment]
> @dancer2090 I was testing the updates that you made in your last commit, and there's only one last thing that we should change before merging this PR.
> The elements that appear in the `Dashboards` section from the footer component should be loaded dynamically, similarly to what we have in the navigation bar.
> Otherwise, when you test it, e.g. in staging, you'll have links pointing to pages that don't exist. Moreover, every time that one of the featured dashboards is renamed, removed, or created, the code would have to be updated, which isn't the right approach.
> 
> We're almost there!

@pablopareja fixed https://github.com/resource-watch/resource-watch/pull/1179/commits/5ddad93c3b4446b6d80de5f4c2168d83f3d4dba1

**Comment by pablopareja** [comment]
Thanks @dancer2090, I'll merge the PR then.
Good job 👏

**Pull request merged**